app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -195,7 +196,14 @@ def create_new_pvc(volume_name, capacity, pvc_name)
                          :requests      => { :storage => capacity + "G" })
     end
 
+    def delete_vm_pvcs(vm_uuid)
+      find_vm_by_uuid(vm_uuid).volumes.each do |volume|
+        client.pvcs.delete(volume.info) if volume.type == "persistentVolumeClaim"
```

Comment:
perhaps line 201 should be surrounded with try-catch to avoid any exception being thrown so all of the pvcs will be tried for deletion, and if failed, to log an appropriate log (The PVC #{pvc_name} that belongs to VM #{vm_name} couldn't be delete due to #{e.message}) or any other message that explains the issue and what needed a further cleanup